src/res/translation/translation_zh_CN.ts

```diff
@@ -0,0 +1,3072 @@
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientsettingsdlg.cpp" line="217"/>
+        <source>If the buffer delay settings are disabled, it is prohibited by the audio driver to modify this setting from within the software. On Windows, press the ASIO Device Settings button to open the driver settings panel. On Linux, use the Jack configuration tool to change the buffer size.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientsettingsdlg.cpp" line="226"/>
+        <source>Sound card driver settings</source>
+        <translation>声卡驱动选项</translation>
+    </message>
+    <message>
+        <location filename="../../clientsettingsdlg.cpp" line="227"/>
+        <source>This opens the driver settings of your sound card. Some drivers allow you to change buffer settings, others like ASIO4ALL let you choose input or outputs of your device(s). More information can be found on jamulus.io.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientsettingsdlg.cpp" line="232"/>
+        <source>Opens the driver settings. Note: </source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientsettingsdlg.cpp" line="233"/>
+        <source> currently only supports devices supporting a sample rate of </source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientsettingsdlg.cpp" line="235"/>
+        <source>Hz. You will not be able to select a driver/device which doesn&apos;t. For more help see jamulus.io.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientsettingsdlg.cpp" line="250"/>
+        <source>ASIO Device Settings push button</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientsettingsdlg.cpp" line="254"/>
+        <source>Skin</source>
+        <translation>皮肤</translation>
+    </message>
+    <message>
+        <location filename="../../clientsettingsdlg.cpp" line="254"/>
+        <source>Select the skin to be used for the main window.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientsettingsdlg.cpp" line="256"/>
+        <source>Skin combo box</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientsettingsdlg.cpp" line="259"/>
+        <source>Language</source>
+        <translation>语言</translation>
```

Comment:
I'd propose to add (Lang) here to ensure that users who selected the wrong language find the correct button. Not sure if we're already doing that for other languages?

Reply:
It doesn't change the language right away (need a restart) so I guess it's okay. I can also add that if you still think it's needed. btw I also suggest the use of the 2nd parameter (if I remember it correctly) of `tr()` so you can add comment for the source string and translators will be able to see that :D